Base/usr/share/man/man1/sh-language.md

```diff
@@ -0,0 +1,133 @@
+## Name
+
+The Shell Command Language
+
+## Introduction
+
+The shell operates according to the following general steps:
```

Comment:
How is this a (1) page? Sounds like `sh`(5) to me...

Reply:
A...file format?
I put it in (1) because I had zero ideas where it would go, so I decided to plop it next to the shell manpage itself.

I guess I can see that...sure, why not.